apps/examples/webserver_test/webserver_test_util.h

```diff
@@ -0,0 +1,248 @@
+#define HTTP_HTTP_VERSION_09      9
+#define HTTP_HTTP_VERSION_10      10
+#define HTTP_HTTP_VERSION_11      11
+
+#define HTTP_CONTENT_LENGTH       0
+#define HTTP_CHUNKED_ENCODING     1
+
+#define HTTP_CONF_MAX_CLIENT                    16
+#define HTTP_CONF_CLIENT_STACKSIZE              8192
+#define HTTP_CONF_MIN_TLS_MEMORY                80000
+#define HTTP_CONF_SOCKET_TIMEOUT_MSEC           50000
+#define HTTP_CONF_SERVER_MQ_MAX_MSG             10
+#define HTTP_CONF_SERVER_MQ_PRIO                50
+#define HTTP_CONF_SERVER_SIGWAKEUP              18
+
+#define HTTP_CONF_MAX_REQUEST_LENGTH            4096
+#define HTTP_CONF_MAX_REQUEST_LINE_LENGTH       256
+#define HTTP_CONF_MAX_REQUEST_HEADER_URL_LENGTH 128
+#define HTTP_CONF_MAX_URL_QUERY_LENGTH          64
+#define HTTP_CONF_MAX_URL_PARAMS_LENGTH         256
+#define HTTP_CONF_MAX_KEY_LENGTH                32
+#define HTTP_CONF_MAX_VALUE_LENGTH              256
+#define HTTP_CONF_MAX_DIVIDED_PATH_LENGTH       32
+#define HTTP_CONF_MAX_SLASH_COUNT               32
+#define HTTP_CONF_MAX_QUERY_HANDLER_COUNT       64
+#define HTTP_CONF_MAX_ENTITY_LENGTH             2048
+
+#define HTTP_ERROR_400            "Bad Request"
+#define HTTP_ERROR_404            "Not Found"
+#define HTTP_ERROR_500            "Internal Server Error"
+
+#define WS_TEST_CONF_MAX_URL_SIZE       50
+#define WS_TEST_CONF_MAX_ENTITY_SIZE    2048
+#define WS_TEST_CONF_MAX_PHRASE_SIZE    50
+#define WS_TEST_CONF_MAX_MESSAGE_SIZE   2100
+
+#define WS_TEST_MODE_GET                     0
+#define WS_TEST_MODE_POST                    1
+#define WS_TEST_MODE_PUT                     2
+#define WS_TEST_MODE_DELETE                  3
+
+#define HTTPSTATUS_NONE            0
+#define HTTPSTATUS_OK              1
+#define HTTPSTATUS_MOVED           2
+#define HTTPSTATUS_ERROR           3
+
+#define ISO_nl                     0x0a
+#define ISO_cr                     0x0d
+#define ISO_space                  0x20
+
+#define WS_TEST_MAXHOSTNAME 128
+
+static const char HTTP11[] = "HTTP/1.1";
+static const char HTTPHOST[] = "host: ";
+static const char HTTPGET[] = "GET ";
+static const char HTTPPOST[] = "POST ";
+static const char HTTPPUT[] = "PUT ";
+static const char HTTPDELETE[] = "DELETE ";
+static const char HTTPCRNL[] = "\r\n";
+
+static const char HTTPFORM[] = "Content-Type: application/x-www-form-urlencoded";
+static const char HTTPCONTENTSIZE[] = "Content-Length: ";
+static const char HTTPCHUNKED[] = "Transfer-Encoding: chunked";
+static const char TRAILERHEADER[] = "Trailer: Expires";
+static const char EXPIRES[] = "Expires: Fri, 21 Oct 2022 07:28:00 GMT";
+
+static const char HEADERFIELD_CONNECT[] = "Connection";
+static const char HEADERFIELD_CLOSE[] = "close";
+static const char HEADERFIELD_KEEPALIVE[] = "keep-alive";
+static const char HEADERFIELD_USERAGENT[] = "User-Agent";
+static const char HEADERFIELD_TINYARA[] = "TinyARA";
+
+#define PRNT(...) \
+	do { \
+		printf("[WS_TEST]%s():%d: ", __func__, __LINE__); \
```

Comment:
You can integrate those 3 code to single line code. please check following example.
```
#define PRNT(fmt, args...) printf("[WS_TEST]%s():%d" fmt, __FUNCTION__, __LINE__, ##args)
```